src/main/webapp/assets/js/util/brooklyn-utils.js

```diff
@@ -175,17 +175,22 @@ define([
     };
 
     Util.logout = function logout() {
-        $.ajax({
-            type: "POST",
-            dataType: "text",
-            url: "/v1/logout",
-            success: function() {
-                window.location.replace("/");
-            },
-            failure: function() {
-                window.location.replace("/");
-            }
-        });
+        var ua = window.navigator.userAgent;
+        if (ua.indexOf("MSIE ") > 0 || ua.indexOf(" Edge/") > 0 || ua.indexOf(" Trident/") > 0) {
+            $.ajax({
+                async: false,
+                type: "GET",
+                dataType: "text",
+                url: "/v1/logout"
+            });
+            document.execCommand('ClearAuthenticationCache', 'false');
```

Comment:
To avoid the (potentially brittle) user agent check - it will fail but it doesn't matter when in a try-catch block. And when it works it will do its thing.